scripts/protocgen.sh

```diff
@@ -27,11 +27,11 @@ done
 
 # command to generate docs using protoc-gen-doc
 buf protoc \
-     -I "proto" \
```

Comment:
Next time just **don't change anything that is not in the PR's primary scope**.

Please.